modules/cnf-troubleshooting-common-ptp-operator-issues.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-troubleshooting-common-ptp-operator-issues_{context}"]
+= Troubleshooting common PTP Operator issues
+
+Troubleshoot common problems with the PTP Operator by performing the following steps.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator on a bare-metal cluster with hosts that support PTP.
+
+.Procedure
+
+. Check the operator and operands are successfully deployed in the cluster.
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ptp
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            READY   STATUS    RESTARTS   AGE
+linuxptp-daemon-5j265           3/3     Running   0          24h
+linuxptp-daemon-s72q2           3/3     Running   0          24h
+ptp-operator-6b8dcbf7f4-czff8   1/1     Running   0          10d
+----
++
```

Comment:
I might add a line of text, probably between this output and the note, mentioning how you use these values later.

Reply:
I've updated the preamble and command. Command now includes the `-o wide` switch so that it's clear that each `linuxpto-daemon` corresponds to a particular node.